docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -41,35 +31,53 @@ This feature provides the necessary mechanisms to enforce access controls when m
 * Clients
 * Roles
 
-It allows you to manage permissions for all resources of a given resource type, such as all users in a realm, or
+You can manage permissions for all resources of a given resource type, such as all users in a realm, or
 for a specific realm resource, such as a specific user or set of users in the realm.
 
-Each realm resource supports a well-defined set of management operations, or actions, that can be performed on them,
-such as view, manage, and resource-specific operations such as view-members if you take groups as an example.
+==== Understanding the scopes of access
+
+Each realm resource supports a well-defined set of management operations, or scopes, that can be performed on them,
+such as `view`, `manage`, and resource-specific operations such as `view-members`, if you take groups as an example.
+
+When managing permissions, you are selecting a set of one or more scopes from a resource type to allow realm administrators
+to perform specific operations on a resource type. For instance, granting a `view` scope will give access to realm administrators
+to list, search, and view, a realm resource. On the other hand, the `manage` scope will allow administrators to perform updates
+and deletes on them.
+
+The scopes are completely independent of each other. If you give access to `manage` a realm resource it does not mean the
+`view` scope is granted automatically. There is no transitive dependency between scopes. Although this might impact the
```

Comment:
```suggestion
The scopes are completely independent of each other. If you give access to `manage` a realm resource, that does not mean the
`view` scope is granted automatically. No transitive dependency exists between scopes. Although this might impact the
```